src/main/java/org/jabref/logic/util/BracketedExpressionExpander.java

```diff
@@ -0,0 +1,1356 @@
+package org.jabref.logic.util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Scanner;
+import java.util.StringJoiner;
+import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jabref.logic.formatter.Formatters;
+import org.jabref.logic.formatter.casechanger.Word;
+import org.jabref.logic.layout.format.RemoveLatexCommandsFormatter;
+import org.jabref.model.cleanup.Formatter;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.FieldName;
+import org.jabref.model.entry.Keyword;
+import org.jabref.model.entry.KeywordList;
+import org.jabref.model.strings.StringUtil;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * @author saulius
```

Comment:
We're trying to get rid of all the `@author` annotations, so please remove this. Thanks to git, this code will be attributed to you anyway via your commit. No need to write it into the source.

Reply:
I bet this is the "user friendly" Eclipse that did it :). Removed. Will push update shortly.